backend/services/esweService.ts

```diff
@@ -227,7 +248,53 @@ export default class EsweService {
         log.info(`Offender record not found in Curious.`)
         return createFlaggedContent(DEFAULT_GOALS)
       }
+      log.error(`Failed to get learner goals. Reason: ${e.message}`)
+    }
+    return createFlaggedContent(null)
+  }
+
+  async getLearnerEducation(nomisId: string): Promise<CurrentCoursesEnhanced> {
+    if (!app.esweEnabled) {
       return createFlaggedContent(null)
     }
+
+    const compareByDateAsc = (a, b) =>
+      compareByDate(parseDate(a.learningPlannedEndDate), parseDate(b.learningPlannedEndDate), false)
+
+    try {
+      const context = await this.systemOauthClient.getClientCredentialsTokens()
+      const courses = await this.curiousApi.getLearnerEducation(context, nomisId)
```

Comment:
Do we have a test that verifies that the curiousApi.getLearnerEducation function gets called with the correct context and nomisId?